src/chrome/internal/stackTraces/formatCallFrameDescription.ts

```diff
@@ -0,0 +1,25 @@
+import * as path from 'path';
+import { DebugProtocol } from 'vscode-debugprotocol';
+import { LoadedSourceCallFrame } from './callFrame';
+import { functionDescription } from './callFramePresentation';
+
+/** The clients can requests the stack traces frames descriptions in different formats.
+ * We use this function to create the description for the call frame according to the parameters supplied by the client.
+ */
+export function formatCallFrameDescription(callFrame: LoadedSourceCallFrame, formatArgs?: DebugProtocol.StackFrameFormat): string {
```

Comment:
Should this be a method on LoadedSourceCallFrame?

Reply:
Moved to CallFramePresentation